### PR TITLE
Fix wso2/micro-integrator/issues/41

### PR DESCRIPTION
**Pull request by malakaganga**
## Purpose
Fixes:  https://github.com/wso2/micro-integrator/issues/41

### CODE REVIEW

File: distribution/src/conf/wso2server.sh [hunk 1]
```diff
@@ -237,10 +237,13 @@ do
...
     fi
 done
-for t in "$CARBON_HOME"/../lib/commons-lang*.jar
+for t in "$CARBON_HOME"/wso2/lib/commons-lang*.jar
 do
     CARBON_CLASSPATH="$CARBON_CLASSPATH":$t
 done
+
+
+
```
Comment (a5anka) [inline distribution/src/conf/wso2server.sh]: Do we need all these extra lines?

File: p2-profile/pom.xml [hunk 2]
```diff
@@ -81,6 +81,9 @@
                                 <featureArtifactDef>
                                     org.wso2.carbon.mediation:org.apache.synapse.wso2.feature:${carbon.mediation.version}
                                 </featureArtifactDef>
+                                <featureArtifactDef>
+                                    org.wso2.carbon.crypto:org.wso2.carbon.crypto.feature:1.0.3
```
Comment (a5anka) [inline p2-profile/pom.xml]: Let's not hard code the version here.

**Comment by a5anka** [comment]
Let's try to use a meaningful title for PRs. The one used in this one is not readable for humans (maybe readable for machines :) ).

File: distribution/src/conf/wso2server.sh (hunk 1 above)
Comment (malakaganga) [reply distribution/src/conf/wso2server.sh]: Fixed with : https://github.com/wso2/micro-integrator/pull/90

**Pull request merged**